src/wormhole/Governance.sol

```diff
@@ -27,8 +42,22 @@ contract Governance {
         }
     }
 
-    struct GovernanceMessage {
-        uint16 chainId;
+    /*
+    * @dev General purpose governance message to call arbitrary methods on a governed smart contract.
+    *      This message adheres to the Wormhole governance packet standard: https://github.com/wormhole-foundation/wormhole/blob/main/whitepapers/0002_governance_messaging.md
+    *      The wire format for this message is:
+    *      - module - 32 bytes
+    *      - action - 1 byte
+    *      - chain - 2 bytes
+    *      - governanceContract - 20 bytes
+    *      - governedContract - 20 bytes
+    *      - callDataLength - 2 bytes
+    *      - callData - `callDataLength` bytes
+    */
+    struct GeneralPurposeGovernanceMessage {
+        bytes32 module;
```

Comment:
I'm not a big fan of putting this in the struct. Could we just do it in the encoding/decoding code?